aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -862,109 +872,116 @@ def _collect_entity_queries(
     node_ids_to_be_exported: Set[int],
     entities_starting_set: DefaultDict[str, Set[str]],
     node_pk_2_uuid_mapping: Dict[int, str],
-    silent: bool,
+    progress_context: ProgressContext,
     include_comments: bool = True,
     include_logs: bool = True,
 ) -> Dict[str, orm.QueryBuilder]:
     """Gather partial queries for all entities to export."""
     # pylint: disable=too-many-locals
-    # Progress bar initialization - Entities
-    progress_bar = get_progress_bar(total=1, disable=silent)
-    progress_bar.set_description_str('Initializing export of all entities', refresh=True)
-
     given_log_entry_ids = set()
     given_comment_entry_ids = set()
     all_fields_info, _ = get_all_fields_info()
 
-    # Universal "entities" attributed to all types of nodes
-    # Logs
-    if include_logs and node_ids_to_be_exported:
-        # Get related log(s) - universal for all nodes
-        builder = orm.QueryBuilder()
-        builder.append(
-            orm.Log,
-            filters={'dbnode_id': {
-                'in': node_ids_to_be_exported
-            }},
-            project='uuid',
-        )
-        res = set(builder.all(flat=True))
-        given_log_entry_ids.update(res)
+    total = 1 + ((1 if include_logs else 0) + (1 if include_logs else 0) if node_ids_to_be_exported else 0)
```

Comment:
that should probably be
```suggestion
    total = 1 + ((1 if include_logs else 0) + (1 if include_comments else 0) if node_ids_to_be_exported else 0)
```

Reply:
by the way, if there are no node ids to be exported, what is the `1` for?

Reply:
oops, yeh I was having an issue without the 1, but I will re-check